modules/cert-manager-self-signed-create.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-self-signed.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-self-signed-create_{context}"]
+= Bootstrapping a CA issuer
+
+// TODO
+You can use a self-signed cluster issuer to bootstrap a CA issuer.
+
+// TODO: if this procedure is just for bootstrapping a CA issuer, then what do we want to show what do you do afterwards? need to add about certs that use that?
+
+.Prerequisites
+
+* You have installed the {cert-manager-operator}
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a namespace for the CA issuer.
+
+.. Create a YAML file that defines the `Namespace` object:
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ca-issuer-namespace <1>
+----
+<1> Specify the namespace for the CA issuer.
++
+// TODO: any particular ns name recommended?
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f namespace.yaml
+----
+
+. Create a self-signed cluster issuer to bootstrap the CA issuer.
+
+.. Create a YAML file that defines the `ClusterIssuer` object:
++
+.Example `self-signed-cluster-issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: selfsigned-issuer
+spec:
+  selfSigned: {}
+----
++
+// TODO: Taken from https://github.com/openshift/cert-manager-operator/blob/master/test/e2e/testdata/self_signed/cluster_issuer.yaml, does it need any other values?
```

Comment:
@thejasn @swghosh Is it correct that this selfSigned field is empty?

Reply:
Yup, that is correct. Self-signed issuers should have `selfSigned: {}` in it's spec.

Reply:
Thanks!